draft-ietf-quic-recovery.md

```diff
@@ -834,19 +834,18 @@ DetectLostPackets(largest_acked):
   foreach (unacked < largest_acked.packet_number):
     time_since_sent = now() - unacked.time_sent
     delta = largest_acked.packet_number - unacked.packet_number
-    if (time_since_sent > delay_until_lost):
-      lost_packets.insert(unacked)
-    else if (delta > reordering_threshold)
-      lost_packets.insert(unacked)
+    if (time_since_sent > delay_until_lost ||
+      delta > reordering_threshold):
```

Comment:
I think the d needs to line up with the 't' of time_since_sent based on looking at OnPacketAcked.

The current indention is confusing, since the delta > reordering_threshold blends into the code that follows.

Reply:
Got it. Fixed.